### ISSUE TITLE
Duplicate Folder in Views

### ISSUE THREAD

**Issue by Nostang3** [puregallery/Pure-Gallery-Support#104]
I'm not sure how it happens but I got one folder that will show up twice in Album or Library view. It's a hidden folder. I tried force closing app, clearing cache, running media scan but it still persists. I'm hesitant to delete data to fix it but will if I have too.

**Comment by Nostang3** [comment]
It's detecting the emulated path and the sdcard path. 

> /sdcard/Albums/Album
> /storage/emulated/0/Albums/Album

**Comment by puregallery** [comment]
Which OS version is this on?
Btw....am back on the project now. Sorry for the hiatus.

**Comment by Nostang3** [comment]
Android 9 (Pie)

**Comment by puregallery** [comment]
nice...so you finally updated from Nougat huh.
Did you see this on the pixel 2?

**Comment by Nostang3** [comment]
Yeah,finally got past N. I was worried but with edxposed and magisk installing pretty easily according to xda, I went for it.

I skipped Pixel 2 and went straight to a 3a from my OG pixel. I wanted a 3 but they were like 300 more.
